db/migrate/20230522194044_move_concurrent_requests_settings_to_vmware.rb

```diff
@@ -0,0 +1,40 @@
+class MoveConcurrentRequestsSettingsToVmware < ActiveRecord::Migration[6.1]
+  class SettingsChange < ActiveRecord::Base
+    serialize :value
+  end
+
+  def up
+    # capture_batch_size
+    #   introduced d1b21ac5444ab07c 10/2022
+    #   only currently populated for vmware
+    #
+    # merged into concurrent_requests
+    #
+    # rename "/ems/ems_*/capture_batch_size" =>
+    #        "/ems/ems_*/concurrent_requests/realtime"
+    # update_all("key = replace(key, 'capture_batch_size', 'concurrent_requests/realtime')")
```

Comment:
I don't understand this comment - it says to update, but instead the code below deletes?